docs/faq.md

```diff
@@ -73,23 +73,17 @@ If Ollama is run as a macOS application, environment variables should be set usi
 
 ### Setting environment variables on Linux
 
-If Ollama is run as a systemd service, environment variables should be set using `systemctl`:
+If Ollama is run as a systemd service, environment variables should be set via the file specified in the `EnvironmentFile` option of the systemd unit:
 
-1. Edit the systemd service by calling `systemctl edit ollama.service`. This will open an editor.
-
-2. For each environment variable, add a line `Environment` under section `[Service]`:
+1. Edit the file `/etc/ollama_serve.conf`:
```

Comment:
config file path is better to `/etc/ollama/serve.conf`

Reply:
sure, made it that way